src/style.css

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  align-items: center;
+  justify-content: center;
+  /* width: 140px;
+  height: 160px; */
+  background-color: #fff;
+  display: block;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
Need to fix it
<img width="1081" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/d07e7fd6-1689-4c01-a52a-7a3c955f758b">